### PR TITLE
Minor files changes and add api endpoints

### LINKED ISSUE #7: Set version to 0.1.0 everywhere

### PR DESCRIPTION
**Pull request by goodstemy**
Change version of the agent to 0.1.0. Add command `web-dev` to run api from Makefile. Replace redis url with env `REDIS_URL` variable. Add four new endpoints (misspelling in commit name): 

- transaction detail
- accounts list of transaction and block
- account detail
- account transactions list

Add api reference .md file.

### CODE REVIEW

File: web/api.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# API REFERENCE
+
+GET: `/api/blocks` - Get all blocks
+GET: `/api/block/{blockNumber}` - Get block detail
+GET: `/api/transactions/{blockNumber}` - Get transactions list
+GET: `/api/transaction/{blockNumber}/{txHash}` - Get transaction detail
```
Comment (chebykin) [inline web/api.md]: we don't need block number here

File: web/api.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+GET: `/api/accounts/{blockNumber}/{txHash}` - Get list of accounts
```
Comment (chebykin) [inline web/api.md]: duplication

File: web/api.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+GET: `/api/account/{address}` - Get account detail
+GET: `/api/account-tx/{address}` - Get transactions list in account
```
Comment (chebykin) [inline web/api.md]: not restful

**Comment by chebykin** [comment]
Also use always plural entities in api path, like transactions, accounts, blocks.

File: web/api.md (hunk 3 above)
Comment (chebykin) [reply web/api.md]: change it to /api/accounts/{account}/transactions

File: web/api.md (hunk 1 above)
Comment (goodstemy) [reply web/api.md]: Why? I don't understand you. It's just a description of URL.

File: web/api.md (hunk 2 above)
Comment (goodstemy) [reply web/api.md]: Can you please tell the lines?

File: web/api.md (hunk 3 above)
Comment (goodstemy) [reply web/api.md]: Did you tell about `/api/account-tx/` URL?

File: web/api.md (hunk 1 above)
Comment (goodstemy) [reply web/api.md]: Block number is needed to get transaction detail from redis. 
Example from https://github.com/kenigtech/keksplorer/blob/master/DEVELOPERS.md
> Hash block_tx:150000:0x000000000:detail Details of tx

Comment (chebykin) [reply web/api.md]: To fetch transactions use a wildcard search

**Comment by chebykin** [comment]
@goodstemy this not closes #7. please, remove this changes form the PR and create a separate one.

**Comment by chebykin** [comment]
api endpoints are listed in a top-level README file, so no need in web/api.md

**Comment by goodstemy** [comment]
Do I need to close this PR and create another one with closes #7?

**Comment by chebykin** [comment]
no, just extract the logic required for #7

**Comment by goodstemy** [comment]
I revert commit that are have #7 logic and change title. Is that right?

**Pull request merged**